cloud-azure/src/main/resources/definitions/azure-credential.json

```diff
@@ -57,6 +59,58 @@
       "type": "String",
       "optional": true,
       "sensitive": false
+    },
+    {
+      "name": "subscriptionId",
+      "type": "String",
+      "sensitive": false
+    },
+    {
+      "name": "tenantId",
```

Comment:
should these be optional? new credentials won't have these fields.

Reply:
it is the old one, the new one is optional